spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/pom.xml

```diff
@@ -18,6 +18,14 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-gcp-starter-data-spanner</artifactId>
 		</dependency>
+
+		<!-- Used to test that multiple Spring Data modules can be used together -->
```

Comment:
Since people will used this sample as a starting point potentially, let's try to keep this out of the pom.xml.
If that means we're not testing for this scenario automatically, that's fine.

Reply:
I think we need a separate sample to demonstrate and test multi-data modules support.

Reply:
moved. done.